webknossos-datastore/app/com/scalableminds/webknossos/datastore/datavault/HttpsDataVault.scala

```diff
@@ -133,6 +147,6 @@ class HttpsDataVault(credential: Option[DataVaultCredential], ws: WSClient) exte
 }
 
 object HttpsDataVault {
-  def create(remoteSourceDescriptor: RemoteSourceDescriptor, ws: WSClient): HttpsDataVault =
-    new HttpsDataVault(remoteSourceDescriptor.credential, ws)
+  def create(remoteSourceDescriptor: RemoteSourceDescriptor, ws: WSClient, dataStoreHost: String): HttpsDataVault =
```

Comment:
please add a comment here explaining what this is for. It’s a pretty strong deviation from before, where the DataVaults and VaultPaths didn’t have to know anything about what a datastore is. I understand why this is still the best place, but we should explain to future readers of this code.
Also maybe explain that dataStoreHost is supposed to be not a remote host, but the “self” host. Maybe you can find better words for that than me…